### PR TITLE
rename depth_to_pointcloud to depthimage_to_pointcloud2

### LINKED ISSUE #35: Rename depth_to_pointcloud -> depthimage_to_pointcloud2
As @mikaelarguedas rightly points out, the name of the 'depth_to_pointcloud' package is inconsistent and not entirely accurate.  We should rename it to "depthimage_to_pointcloud2", which is both accurate and consistent with other names we are using.

### PR DESCRIPTION
**Pull request by mikaelarguedas**
To be merged after beta2 announcement

Fixes #35

### CODE REVIEW

**Comment by mikaelarguedas** [comment]
ready for review

**Review by dirk-thomas — APPROVED** [review]
lgtm

**Comment by mikaelarguedas** [comment]
@nuclearsandwich FYI: that means the the previous deb will not be used for anything else than beta2 (we may decide it to remove it from the repos for the next release) and that a new deb will be generated for this package next time we bloom.

**Comment by nuclearsandwich** [comment]
> @nuclearsandwich FYI: that means the the previous deb will not be used for anything else than beta2 (we may decide it to remove it from the repos for the next release) and that a new deb will be generated for this package next time we bloom.

👍

File: turtlebot2_cartographer/launch/turtlebot_carto_3d.py [hunk 1]
```diff
@@ -35,11 +35,12 @@ def launch(launch_descriptor, argv):
         name='astra_camera_node',
         exit_handler=restart_exit_handler,
     )
-    package = 'depth_to_pointcloud'
+    package = 'depthimage_to_poincloud2'
```
Comment (clalancette) [inline turtlebot2_cartographer/launch/turtlebot_carto_3d.py:38]: This should be "depthimage_to_pointcloud2"

File: turtlebot2_cartographer/launch/turtlebot_carto_3d.py [hunk 2]
```diff
@@ -35,11 +35,12 @@ def launch(launch_descriptor, argv):
...
     ld.add_process(
         cmd=[
-            get_executable_path(package_name=package, executable_name='depth_to_pointcloud_node')],
-        name='depth_to_pointcloud_node',
+            get_executable_path(
+                package_name=package, executable_name='depthimage_to_poincloud2_node')],
+        name='depthimage_to_poincloud2_node',
```
Comment (clalancette) [inline turtlebot2_cartographer/launch/turtlebot_carto_3d.py:43]: Same in this block.

**Comment by mikaelarguedas** [comment]
typo fixed in https://github.com/ros2/turtlebot2_demo/commit/b9d771b52fa307c08dcbab397a08d17f979982d9
thanks @clalancette for catching that!

**Pull request merged**